visualization/app/codeCharta/ui/export3DMapButton/export3DMapButton.component.scss

```diff
@@ -0,0 +1,27 @@
+cc-export-threed-map-button {
+	margin-left: 8px;
+	.mat-flat-button.mat-accent,
+	.mat-raised-button.mat-accent,
+	.mat-fab.mat-accent,
+	.mat-mini-fab.mat-accent {
+		background-color: rgb(0, 150, 136);
+	}
+	.mat-mini-fab {
+		height: 25px;
+		width: 25px;
+	}
+	.button-icon {
+		font-size: 16px;
+		line-height: 1.3;
+	}
+	span {
+		padding: 0;
+		margin: 0;
+	}
+	.mat-mini-fab .mat-button-wrapper {
+		height: 25px;
```

Comment:
width and height are redundant with above `.mat-mini-fab` and can be removed here, aren't they?

Reply:
Resolved in the next push